pilot/test/integration/kubernetes_external_name_services.go

```diff
@@ -14,6 +14,7 @@
 
 package main
 
+/*
```

Comment:
Do we need to remove this as well ?

Reply:
It would sufficient to just disable the test in https://github.com/istio/istio/pull/3039/files/4b49946edf5f7af4ecb9a18f8eb45c82c1c28a25#diff-a2a49898041cb954e7bfb2ce9b0f52b0.